### PR TITLE
Remove variable parameters from SavePlayerScore action.

### PR DESCRIPTION
**Pull request by D8H**
image

### CODE REVIEW

File: Extensions/Leaderboards/leaderboardstools.ts [hunk 1]
```diff
@@ -233,30 +227,25 @@ namespace gdjs {
                 response.statusText
               );
               scoreSavingState.setError(errorCode);
-              errorVar.setString(errorCode);
               return;
             }
 
             scoreSavingState.closeSaving();
 
             return response.text().then(
```
Comment (AlexandreSi) [inline Extensions/Leaderboards/leaderboardstools.ts:235]: This can be removed I think

**Comment by D8H** [comment]
I modified the example not to use these parameters and check the "submit" button after a submission and another run.
* https://github.com/GDevelopApp/GDevelop-examples/pull/256/commits/7dedfb7312d2e57f1792ce16881dfc87e02b4772

File: Extensions/Leaderboards/leaderboardstools.ts (hunk 1 above)
Comment (D8H) [reply Extensions/Leaderboards/leaderboardstools.ts:235]: There is condition to check a failure. An expression can be added later to give more details about the error, but this is not mandatory.

Comment (AlexandreSi) [reply Extensions/Leaderboards/leaderboardstools.ts:235]: That's ok with me

**Review by AlexandreSi — APPROVED** [review]

File: Extensions/Leaderboards/leaderboardstools.ts (hunk 1 above)
Comment (4ian) [reply Extensions/Leaderboards/leaderboardstools.ts:235]: Yes I think it's good to keep 👍

**Review by 4ian — APPROVED** [review]

**Pull request merged**